packages/psss/src/containers/Tables/ArchiveTable.js

```diff
@@ -61,36 +65,45 @@ const createColumns = isForMultipleCountries => [
     key: 'id',
     sortable: false,
     CellComponent: AlertMenuCell,
-    width: '70px',
+    // width: '70px',
+    width: '45px',
   },
 ];
 
-export const ArchiveTable = React.memo(({ countryCode }) => {
-  const { isLoading, isFetching, error, data, order, orderBy, handleChangeOrderBy } = useTableQuery(
-    'archive',
-  );
+const getColumns = isSingleCountry =>
+  isSingleCountry ? archiveColumns : [countryColumn, ...archiveColumns];
+
+const ArchiveTableComponent = React.memo(({ countryCode, countryCodes }) => {
```

Comment:
I foresee the ArchiveTable, ArchiveTabView etc being very different from the AlertsTabView etc so I think it's great that these are seperate templates.

Reply:
Agreed! Any similarities could be extracted in shared components. They have some obvious similarities (eg shared columns - but not always column widths), but extracting partial similarities would probably make things more complicated than simplify, in this case.